client/src/Component/NavbarComponent/ButtonAppBarCollapse.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { withStyles } from "@material-ui/core/styles";
+import { Menu } from "@material-ui/core";
+import Icon from "@material-ui/core/Icon";
+
+const styles = (theme) => ({
+  buttonCollapse: {
+    [theme.breakpoints.up("sm")]: {
+      display: "none",
+    },
+    margin: "10px",
+    boxShadow: "none",
+  },
+});
+
+class ButtonAppBarCollapse extends React.Component {
```

Comment:
Any particular reason to use a class component here instead of a function component?